python/spacewalk/satellite_tools/reposync.py

```diff
@@ -2546,12 +2547,13 @@ def upload_patches(self, notices):
         return processed_updates
 
     def errata_needs_update(
-        self, existing_errata, new_errata_version, new_errata_changedate
+        self, existing_errata, new_errata_version, new_errata_changedate, packages=[]
```

Comment:
A list is not a good default argument. Please use `None` and check for `None` inside the function body to set it to an empty list instead ([style guide](https://google.github.io/styleguide/pyguide.html#212-default-argument-values)).